### PR TITLE
#3 Responsive Layout

### LINKED ISSUE #3: Fix any issues on mobile
Test the UI with mobile wallets. Some users have complained that it doesn't work for them.

There isn't much information on what exactly doesn't work.

### PR DESCRIPTION
**Pull request by nipa85**
resolves #3 

Responsive menu for smaller screens. 
Footer always stick to the bottom of the page.

Some changes to swap form, token selector, order view for better appearance on small screens.

Please check the new menu for other changes.

### CODE REVIEW

**Review by vfatouros — APPROVED** [review]
I have reviewed the changes just by checking that resizing to a small window works well now, and there is a new sandwich menu for small screens which looks good. Tested a swap as well. However I am not familiar with CSS so can't speak for the actual code changes.

File: src/components/settings/settings.riot [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
     <div class="dropdown" onclick={ toggleDropDown } id="settings-icon">
         <div class="dropdown-logo">
-            <img src={ Image } alt="settings">
```
Comment (SHA-2048) [inline src/components/settings/settings.riot:5]: I think there is an import that define `Image`. So we can remove it as well now.

**Review by SHA-2048 — APPROVED** [review]

File: src/components/settings/settings.riot (hunk 1 above)
Comment (nipa85) [reply src/components/settings/settings.riot:5]: Of course. I remove it.

**Comment by talegift** [comment]
Token selector should remain to look the same as it looks on master now

https://share.getcloudapp.com/xQuAwQNv
vs
https://share.getcloudapp.com/jkuDX88E

**Comment by vfatouros** [comment]
This seems ok now (layout is the same as current master, works better on mobile). Tested a swap.

**Comment by SHA-2048** [comment]
All changes here are included in PR #43 after merging and adapting logic to order dashboard. I think this one can be closed.

**Pull request closed without merging**